packages/website/components/Topbar.tsx

```diff
@@ -126,16 +126,26 @@ export const Topbar = () => (
               </a>
             </Link>
           </li>
+          <li className={styles.navListItem}>
+            <a
+              className={styles.navListLink}
+              href="https://v3.f36.contentful.com/"
+            >
+              <Subheading className={styles.navListLink} marginBottom="none">
+                v3
+              </Subheading>
```

Comment:
I think all Subheadings used as in the topbar should not be `h3`
so they need a `as='span'` or something like that

or maybe this works:

```
<li>
  <Subheading
     as="a"
     href="https://v3.f36.contentful.com/"
     className={styles.navListLink} 
     marginBottom="none"
   >
     v3
   </Subheading>
</li>
```